models/user.model.js

```diff
@@ -10,6 +10,8 @@ const UserSchema = new mongoose.Schema(
     phone_number: { type: String, required: true },
     donations: { type: Array, default: [] },
     aadhar_number: { type: String, required: true },
+    isVerified: { type: Boolean, default: false },
+    emailtoken: { type: String },
```

Comment:
since we are using _ separated words, is_verified and email_token are more apt.

Reply:
okay I will make the changes and update PR